.github/workflows/deploy-dev.yaml

```
@@ -58,3 +58,7 @@ jobs:
           cloudflare_account_id: ${{ vars.CLOUDFLARE_ACCOUNT_ID }}
           cloudflare_api_token: ${{ secrets.CLOUDFLARE_API_TOKEN }}
           deploy_env: ${{ vars.DEPLOY_ENV }}
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

actionlint (1.7.4)

input "github_token" is not defined in action "Deploy to Cloudflare" defined at "./.github/actions/cloudflare-deploy". available inputs are "api_base_url", "api_key", "app_base_url", "auth0_audience", "auth0_client_id", "auth0_client_secret", "auth0_domain", "auth0_secret", "cloudflare_account_id", "cloudflare_api_token", "deploy_env", "turbo_api", "turbo_team", "turbo_token"

(action)